config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
+
+  get '/books' => 'book#index', as: 'books'
```

Comment:
I would use the default route generator for resources, in this case is
```ruby
resources :books, only: [:index, :show]
```

Reply:
I considered doing it this way. But it does not work :sweat_smile: